corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -180,6 +188,11 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         var mapping = {
             caption: {
                 update: function (options) {
+                    let parentForm = getParentForm(self);
+                    let oqps = parentForm.displayOptions.oneQuestionPerScreen != undefined && parentForm.displayOptions.oneQuestionPerScreen();
+                    if (!oqps) {
+                        return null;
```

Comment:
I'd thought this would be in `Repeat` - don't question lists also have a header? Or is that stored somewhere different?

Reply:
You're right, Question Lists do have a header. I'm working on accounting for that.